### ISSUE TITLE
Some questions on fine-tuning

### ISSUE THREAD

**Issue by ZeweiLi999** [naoto0804/SynShadow#21]
Hi naoto! 
Thank for your great work! It's really helpful for my work. Can you tell me some details about how to fine-tuning the model from the Synthetic Dataset to a real dataset? It will be useful to my work. Thank for your work again.

**Comment by naoto0804** [comment]
Hi, I'm glad to hear that my old project is helpful!
I used a similar setting (e.g., optimizer, learning rate, #iter ...) for training and fine-tuning, by just copying the recommended training setting for each original paper. My project's primary goal was to prove the usefulness of synthetic data for many (models, tasks, datasets), so I didn't have enough time to carefully tune hyper-parameters for each triplet (model, task, dataset), resulting in the aforementioned single (probably plausible) setting. If you want to enhance the performance on the specific triplet, following standard practices (e.g., early stopping, LR sweep, ...) might probably give a boost.

**Comment by ZeweiLi999** [comment]
Thank for your answer!
